docs/serving/getting-started-knative-app.md

```diff
@@ -29,6 +29,12 @@ adapted for your own application if you have an image of it available on
 [Google Container Registry](https://cloud.google.com/container-registry/docs/pushing-and-pulling),
 or another container image registry.
 
+In case your image is in the local registry, you need to tag it as following first:
+
+```bash
+docker tag local-image dev.local/local-image # use dev.local/local-image in your yamls
+```
+
```

Comment:
It might be helpful to point to the configmap to customize the registry prefix? Or do you think this will be too much information to get started?

Reply:
Yeah, I think this would be too much for this page. Perhaps it'd be a good idea to have a single page documents these small configurations.

Reply:
FYI: I think this new text is also too much info for this context but given that we lack a dedicated page or user task that explains how to deploy an image (ie. from a container image registry), its okay for it to live here for now. 

Ideally, we cover only the common used case in a "getting started" to demonstrate a single path to success (and avoid distracting and/or confusing new uses with a bunch of conditionals). If we did have a dedicated topic about deploying from registries, we would instead just have a pointer to that info. For example: 

`[Learn more about deploying your app, including how to deploy local images.](link-to-deploying-topic.md)`

I will keep an eye on this content and implement the suggested changes when we do get around to authoring a stand-alone "how to deploy apps" topic.